lib/src/domain/settings/popular_news_criterion.dart

```diff
@@ -0,0 +1 @@
+enum PopularNewsCriterion { viewed, shared, emailed }
```

Comment:
jak będziemy mieć już architekturę to wrzuciłbym to do tego samego pliku co klasa dziedzicząca po ChangeNotifierze, to jakby nie patrzeć jakaś forma stanu, co myślisz?

Reply:
nope, jest to domena, później, jeżeli dojdzie jakaś logika to któraś z wartości tego enuma będzie zapisywana w repozytorium i na jej podstawie będzie określane po co bijemy na home_page :)

Reply:
ano rzeczywiście to będzie używane w repo